src/main/java/com/crud/tasks/service/TrelloService.java

```diff
@@ -35,7 +38,7 @@ public CreatedTrelloCardDto createTrelloCard(final TrelloCardDto trelloCardDto)
         ofNullable(newCard).ifPresent(cart -> emailService.send(new Mail(
                 adminConfig.getAdminMail(),
                 SUBJECT,
-                "New card: " + trelloCardDto.getName() + " has been created on your Trello account")));
+                mailCreatorService.buildTrelloCardEmail("New card: " + trelloCardDto.getName() + " has been created on your Trello account"))));
```

Comment:
Jeśli jedynym zmiennym elementem maila jest nazwa karty Trello, to ona powinna być argumentem do metody `MailService::sendCreatedTrelloCardEmail`. Stała część tekstu (`New card: ` i `has been...`) powinna trafić bezpośrednio do szablonu HTML.
Generalnie, po to masz szablon, żeby tam była cała treść, a w kontekście powinny być przekazywane tylko bezpośrednio zmieniające się elementy.